selfdrive/controls/controlsd.py

```diff
@@ -685,19 +686,21 @@ def publish_logs(self, CS, start_time, actuators, lac_log):
     self.pm.send('controlsState', dat)
 
     # carState
-    car_events = self.events.to_msg()
+    if events_changed := (self.events.names != self.events_prev):
```

Comment:
The walrus operator has it's valid uses, but don't think this makes things more clear. Just write this as two lines.

```suggestion
    events_changed = self.events.names != self.events_prev
    if events_changed:
```